http-api/src/lib.rs

```diff
@@ -43,6 +45,33 @@ pub struct Options {
 pub struct Context {
     paths: Paths,
     theme: String,
+    aliases: Arc<RwLock<HashMap<String, Urn>>>,
+}
+impl Context {
+    /// Populates alias map with unique projects' names and their urns
+    async fn populate_aliases(&self, map: &mut HashMap<String, Urn>) -> Result<(), Error> {
+        use radicle_daemon::git::identities::SomeIdentity::Project;
+
+        let storage = ReadOnly::open(&self.paths).expect("failed to read storage paths");
+        let identities = identities::any::list(&storage)?;
+
+        for identity in identities.flatten() {
+            if let Project(project) = identity {
+                let urn = project.urn();
+                let name = (&project.payload().subject.name).to_string();
+
+                tracing::info!("alias {:?} for {:?}", name, urn.to_string());
+
+                if let std::collections::hash_map::Entry::Vacant(e) = map.entry(name.clone()) {
+                    e.insert(urn);
+                } else {
+                    tracing::warn!("alias {:?} exists, skipping", name);
+                }
+            }
+        }
```

Comment:
We don't really care about non-project aliases since we only suppor them for projects, hence you can simplify this code by using `identities::project::list`.

Reply:
Since we currently don't have the `identities::project::list` function, we could maintain this function, stripping the debug statements